packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md

```diff
@@ -364,6 +364,31 @@ Sample user data:
 | Workday ID                                                         | `hasWorkdayUser() ? findWorkdayUser().employeeID : null`                                                                                                 | 123456                  | Check if user has a Workday assignment, and if so, return their Workday employee ID.                                                                                                                                                                                                                                                                                                                   |
 | Active Directory UPN                                               | `hasDirectoryUser() ? findDirectoryUser().managerUpn : null`                                                                                             | bob@okta.com            | Check if user has an Active Directory assignment, and if so, return their Active Directory manager UPN.                                                                                                                                                                                                                                                                                                |
 
+
+## Expressions for custom claims
+
+Okta provides a few expressions that you can only use with custom claims.
+* See [Create claims](/docs/guides/customize-authz-server/create-claims).
+* See [Include app-specific information in a custom claim](/docs/guides/customize-tokens-returned-from-okta/create-app-profile-attribute).
+
+| Syntax           | Definitions                                                             | Examples     |
+| --------         | ----------                                                              | ------------ |
+| `app.$attribute` | `app` refers to the name of the OIDC app.<br>`$attribute` refers to the attribute variable name. | app.id<br>app.clientId<br>app.profile |
+| `access.scope` | `access` refers to the access token requesting the Scopes.<br>`scope` refers to the array of granted Scopes. | access.scope |
+
+### Samples
+
+#### Sample using an app rule
```

Comment:
`Sample using app attributes` instead of `Sample using an app rule`. `rule` has special meaning in Okta, for example - group rules. But it does not apply here.